python/lsst/ts/watcher/rules/cp_verify_bias_alarm.py

```diff
@@ -0,0 +1,255 @@
+# This file is part of ts_watcher.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["CpVerifyBiasAlarm"]
+
+import yaml
+import json
+import lsst.daf.butler as dafButler
+
+from lsst.ts.idl.enums.Watcher import AlarmSeverity
+from lsst.ts import watcher
+
+
+class CpVerifyBiasAlarm(watcher.BaseRule):
+    """Report if cp_verify BIAS tests failed.
+
+    Set alarm severity NONE if the cp_verify tests passed,
+    and issue a WARNING if not.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+
+    Notes
+    -----
+    The alarm name is f"CpVerifyBiasAlarm.{name}:{index}",
+    where name and index are derived from ``config.name``.
+    """
+
+    def __init__(self, config):
+        remote_name = "OCPS"
+        # remote_index = 0: process events from both OCPS's
+        remote_index = 0
+        remote_info = watcher.RemoteInfo(
+            name=remote_name,
+            index=remote_index,
+            callback_names=["job_result"],
+            poll_names=[],
+        )
+        super().__init__(
+            config=config,
+            name=f"CpVerifyBiasAlarm.{remote_info.name}:{remote_info.index}",
+            remote_info_list=[remote_info],
+        )
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            description: Configuration for CpVerifyBiasAlarm
+            type: object
+            properties:
+                name:
+                    description: >-
+                        CSC name and index in the form `name` or `name:index`.
+                        The default index is 0.
+                    type: string
+                number_verification_tests_threshold_bias:
```

Comment:
maybe just `verification_threshold`?

Reply:
I kept the long name because that's the same one that it has in the calibration script. In that script, there's also one of these parameters per calibration type: bias, dark, flat.

Reply:
So, in order to have the same as the calibration script, should we have three parameters here?

Reply:
no, use the short version and it will be the one for whatever calibration the alarm is configured for...